.travis.yml

```diff
@@ -13,6 +13,7 @@ notifications:
 
 install:
   - pip install --quiet -r requirements.txt
+  - pip install --quiet -r requirements.test.txt
```

Comment:
Can we put these new requirements in `requirements.txt` rather than creating a new file? I think `requirements.txt` is only used ever for travis testing.

Reply:
Alright! It makes sense (not only travis, also developer that forks the repo can use requirements.txt. But it makes sense to install the required test packages as a developer)